test/service/test_manifest_async.py

```diff
@@ -301,7 +301,7 @@ def test(self, mock_uuid, mock_helper):
                     with self.subTest(manifest=manifest):
                         expected_status = 410 if manifest is None else 302
                         self.assertEqual(object_key, manifest_url.args['objectKey'])
-                        response = requests.get(manifest_url.url, allow_redirects=False)
+                        response = requests.get(manifest_url, allow_redirects=False)
```

Comment:
I think we still want to wrap the url with str() even though requests.get is able to handle url-like objects.

we should check in with @achave11  i think he brought this up during #3011

Reply:
@amarjandu, that is correct. We use `str(manifest_url)` even though `requests.get` handles furl correctly.